refarch-eai/pom.xml

```diff
@@ -35,7 +35,7 @@
         <findsecbugs-plugin.version>1.13.0</findsecbugs-plugin.version>
 
         <!-- Logging -->
-        <logstash-logback-encoder.version>7.4</logstash-logback-encoder.version>
+        <logstash-logback-encoder.version>8.0</logstash-logback-encoder.version>
```

Comment:
Is think bump to Logstash 8.0 required here? We'd like to keep it at `7.x` because Logstash 8.0 is officially only supported by Logback 1.5 and above. Logback 1.5 is only included in Spring 3.3.x and above. And because our Spring version is bound to the Camel version in the EAI case. We are still stuck at Spring 3.2.x.
Is your feature also possible with Logstash 7.x?

Reply:
didn't see this dependencies, it also works fine with 7.4 in EAI. therefore removed version bump in EAI